pyvista/plotting/plotting.py

```diff
@@ -2646,7 +2699,7 @@ def get_image_depth(self,
         # Convert z-buffer values to depth from camera
         near, far = self.camera.GetClippingRange()
         if self.camera.GetParallelProjection():
-            z_val = (zbuff - near) / (far - near)
+            zval = (zbuff - near) / (far - near)
```

Comment:
Oh wow, thanks!!!

Reply:
the spyder IDE is very nice because it spots all these small issues automatically